amplify-tools-gradle-plugin/src/main/groovy/com/amplifyframework/tools/gradle/plugin/AmplifyTools.groovy

```diff
@@ -0,0 +1,226 @@
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+import groovy.json.JsonSlurper
+import groovy.json.JsonOutput
+
+class AmplifyTools implements Plugin<Project> {
+    void apply(Project project) {
+        def doesNodeExist = true
```

Comment:
Is this boolean necessary? I'd think you should have any task that relies on node (and the CLI for the matter), just run `verifyNode` first and go boom if either is unavailable.